### PR TITLE
Allocation bug

### LINKED ISSUE #187: Allocation chart does not show active allocation
When user has more than one allocation, one active and the other in the future. The allocation chart page shows only the inactive allocaction, that will happen in the future.

### PR DESCRIPTION
**Pull request by tonyaraujop**
Closes #187 

Allocation page:
image

Allocation chart:
image

### CODE REVIEW

File: app/queries/allocations_and_unalocated_users_query.rb [hunk 1]
```diff
@@ -30,6 +30,6 @@ def where
   end
 
   def last_user_allocations
-    Allocation.group(:user_id).maximum(:end_at).values
+    Allocation.where(ongoing: true).group(:user_id).maximum(:end_at).values
```
Comment (adbatista) [inline app/queries/allocations_and_unalocated_users_query.rb:33]: why not use the allocation scope, `.ongoing` ?

File: spec/models/allocation_spec.rb [hunk 2]
```diff
@@ -176,5 +176,13 @@
         expect(subject.days_until_finish).to eq('Finalizado')
       end
     end
+
+    context 'when allocation did not started' do
+      subject { build_stubbed(:allocation, end_at: Time.zone.today + 1.month) }
```
Comment (adbatista) [inline spec/models/allocation_spec.rb:181]: just to let you know; you can also do `1.month.from_now`

File: app/queries/allocations_and_unalocated_users_query.rb (hunk 1 above)
Comment (tonyaraujop) [reply app/queries/allocations_and_unalocated_users_query.rb:33]: I miss that one. Changing right now.

File: spec/models/allocation_spec.rb (hunk 2 above)
Comment (tonyaraujop) [reply spec/models/allocation_spec.rb:181]: Nice.

**Pull request merged**